Plugins/NativeEngine/Source/ShaderCompilerD3D.h

```diff
@@ -96,6 +98,7 @@ namespace Babylon
         }
 
         ShaderCompilerTraversers::IdGenerator ids{};
+        auto cutScope = ShaderCompilerTraversers::ChangeUniformTypes(program, ids);
```

Comment:
uniform bloc is the same for the VS and FS. FS uniforms definition start with an offset in that buffer. For D3D12, this offset must be 16bytes aligned. So, with vec4/mat4, it is. D3D11 uniform definition is different but making it aligned anyway.